docs/quotes/README.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Quotes
+permalink: /quotes
+---
+
+# Quotes
+
+Quotes can ultimately be used for PSI element -> String transformations and vice-versa. For example: 
+
+```kotlin
+ifExpression({ true }) { e ->
+     Transform.replace(
+      replacing = e,
+        newDeclaration = """ if $`(condition)` $then $`else` """.`if`
+       )
+}
+```
+
+Below you can find a comprehensive map between Kotlin's PSI element constructs and various quotes in arrow.meta.quotes. If something's missing here, please let us know [here in our issue tracker](https://github.com/arrow-kt/arrow-meta/issues).
+
+## ClassOrObject (quotes.classorobject)
+|  Quote                                                                                                                        |  Declaration 
+| ------------------------------------------------------------------------------------------------------------------------------| ----------------------------------------------------------------------------------------------------------- |
+| [Class](/docs/apidocs/compiler-plugin/arrow.meta.quotes/classorobject/class.html)                                             | ```"""$`@annotations` $kind $name $`(typeParameters)` $`(params)` : $supertypes"} { $body""".`class` ```    |
```

Comment:
The table looks great! 
We should be using the dokka docs or this is gonna be hard to manually maintain. Have you tried adding this to the kdoc and trying to use the auto-generated table? 

Maintaining this table manually is going to become tedious as the lang changes and some of these which are not type-checked by ank become outdated with those changes. There are many upcoming lang features that may change the scopes and therefore render this table outdated when that happens.

Reply:
@raulraja here's my attempt with autogeneration (both options are still available) let me know what you think:

![Screen Shot 2019-11-22 at 5 01 34 PM](https://user-images.githubusercontent.com/6917498/69454016-886e2800-0d65-11ea-9cce-5bebce76f5ff.png)

Not sure if there's a way to prevent signatures in autogeneration but I can look into it

Reply:
Signature are fine IMO. I discussed with @franciscodr today we should just change those comments below the signature to be the type checked ank code snippets. Are those code snippets with ank or just strings?

Reply:
ah - these have the markdown for code snippets but without ank - let me see if I'm able to use ank with the correct imports.  It is the same case for the README.md for quotes, but they were not formatted in this case.

Reply:
@raulraja Is it possible to run typechecking on all this without having to wrap it in the Transform and meta functions?

Reply:
For example, without the wrappings, there's no way for the code to know what I'm talking about I'm sure:
![Screen Shot 2019-11-25 at 10 48 55 AM](https://user-images.githubusercontent.com/6917498/69529823-487b9080-0f71-11ea-891c-ce04ec1449fc.png)